conf/cassandra.yaml

```diff
@@ -1757,6 +1757,23 @@ drop_compact_storage_enabled: false
 # write_consistency_levels_warned: []
 # write_consistency_levels_disallowed: []
 #
+# Guardrail to warn or fail when writing column values larger than threshold.
+# This guardrail is only applied to the values of regular columns because both the serialized partitions keys and the
+# values of the components of the clustering key already have a fixed, relatively small size limit of 65535 bytes, which
+# is probably lesser than the thresholds defined here.
+# Deleting individual elements of non-frozen sets and maps involves creating tombstones that contain the value of the
+# deleted element, independently on whether the element existed or not. That tombstone value is also guarded by this
+# guardrail, to prevent the insertion of tombstones over the threshold. The downside is that enabling or raising this
+# threshold can prevent users from deleting set/map elements that were written when the guardrail was disabled or with a
+# lower value. Deleting the entire column, row or partition is always allowed, since the tombstones created for those
+# operations don't contain the CQL column values.
+# This guardrail is different to max_value_size. max_value_size is checked when deserializing any value to detect
+# sstable corruption, whereas this guardrail is checked on the CQL layer at write time to reject regular user queries
+# inserting too large columns.
+# The two thresholds default to null to disable.
+# column_value_size_warn_threshold:
```

Comment:
I had a hard time to realise what is the actual unit of size we are supporting here. I guess it is in bytes but it is not obvious from this comment. Maybe we might explicitly mention it?

Reply:
All units, not only bytes. We have marked min unit only for old properties where we couldn't support anything less that the mentioned unit and the larger units. Considering I wrote a very detailed doc about config and what we support I do not think that we should add anything here really? We have it stated what units we support for Data Storage in the docs and we marked those which are exception due to legacy reasons in the yaml. I talk about this doc - https://cassandra.apache.org/doc/trunk/cassandra/new/configuration.html. There is also a blog post. Please let me know if you think there is anything unclear,  I will be making a few updates tomorrow to the mentioned doc and I can add/edit anything that might seem confusing. Thanks

Reply:
Ha....I actually noticed I stated the min unit, but not explicitly in that doc - these are all units supported per type of parameter. Good catch. They are in the ticket and other places but we need them listed explicitly in a table I think. Thanks!

Reply:
Just added a comment about the min unit in the yaml.